website/templates/project/project_base.mako

```diff
@@ -116,7 +116,7 @@
 <%def name="image_meta()">
     <%
         from website import settings
-        return settings.DOMAIN.rstrip('/') + settings.PREPRINTS_ASSETS + 'osf/sharing.png'
+        return settings.DOMAIN.rstrip('/') + settings.STATIC_URL_PATH + '/img/osf-sharing.png'
```

Comment:
I like joins more, but 🤷‍♂️ 
`'{}{}/img/osf-sharing.png'.join(settings.DOMAIN.rstrip('/'), settings.STATIC_URL_PATH)`

Reply:
`.format`, you mean?

Reply:
derp fixed.

Reply:
I mean, I can join it if you really want
```python
In [2]: '/'.join([settings.DOMAIN.rstrip('/'), settings.STATIC_URL_PATH.lstrip('/'), 'img', 'osf-sharing.png'])
Out[2]: 'http://localhost:5000/static/img/osf-sharing.png'
```